.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Release
+
+run-name: release new version by @${{ github.actor }}
+
+on:
+  workflow_dispatch:
+
+env:
+  PIP_DISABLE_PIP_VERSION_CHECK: on
+  PIP_NO_CLEAN: on
+  PIP_NO_INPUT: on
+  PIP_PREFER_BINARY: on
+
+permissions:
+  contents: write
+  packages: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  containerize:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      - name: Setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.9
+
+      - name: Install requirements
+        run: make setup
+
+      - name: Install project
+        run: pipenv install --deploy
```

Comment:
This is our second install, the previous being in the makefile. Is this for a particular purpose? If it's to catch app init issues, note it omits the frontend webpack build so isn't complete coverage.

Reply:
true. fixed in 1ca72e1